<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanWindow</class>
 <widget class="QDialog" name="BanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Users' list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="userLoginBrowser"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type or copy user's name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userLoginLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="banButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ban for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="banTimeComboBox">
       <item>
        <property name="text">
         <string>remove from ban</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24 h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>72 h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>forever</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
